spec/controllers/homepage_controller_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe HomepageController, type: :controller do
+  describe "GET 'index'" do
```

Comment:
It's not necessary to test Rails's built-in functionality (i.e. `.reverse`) but it might be a good idea to test that `@books`, `@movies`, etc. are set as expected (i.e. there's only 10 objects, they're in reverse order, etc.).